src/mmap.rs

```diff
@@ -165,6 +165,7 @@ impl Bytes<MemoryRegionAddress> for GuestRegionMmap {
     fn write(&self, buf: &[u8], addr: MemoryRegionAddress) -> guest_memory::Result<usize> {
         let maddr = addr.raw_value() as usize;
         self.as_volatile_slice()
+            .unwrap()
```

Comment:
Hi, @bonzini , @andreeaflorescu . Please pay attention to the code update here.

In the code base (without this PR), reference of `GuestRegionMmap` has had `get_slice(`) and `as_volatile_slice()`, because `GuestRegionMmap` is de-referenced to `MmapRegion` (https://github.com/rust-vmm/vm-memory/blob/master/src/mmap.rs#L147).

In my first commit, I added `GuestMemoryRegion::as_volatile_slice()`, so what's called here became the new added function. There was no problem because the signature of the 2 `as_volatile_slice()`'s is identical. But when I tried to modify the return type of `GuestMemoryRegion::as_volatile_slice()`, the code here didn't build. That's why I modified this line.

Reply:
Yes, this is okay. The GuestRegionMmap knows that its as_volatile_slice implementation cannot fail.